src/components/organisms/Card/Card/Card.tsx

```diff
@@ -59,6 +80,9 @@ const Card = styled.div<CardProps>`
   & ${CardText} {
     font-size: ${({ styling = 'secondary' }) => textSizes[styling]};
   }
+  & ${CardContent} {
```

Comment:
Rather than setting background colour to `CardContent` i'd set it to `Card` https://github.com/zopaUK/react-components/pull/588/files#diff-352a38919c0c8ed007ece52ccaf4be96R69